client/my-sites/hosting/sftp-card/index.js

```diff
@@ -116,41 +124,96 @@ const SftpCard = ( {
+								'{{p}}A variety of custom plugins and themes may ask you to create specific folders or add files via SFTP however, generally speaking, SFTP is not required for your site to function.{{/p}}',
+							{
+								components: {
+									p: <p />,
+									a: <a href={ FILEZILLA_URL } />,
+								},
+							}
+						) }
+					</Accordion>
+					<Accordion title={ translate( 'How to use SFTP to access your site' ) }>
+						{ translate(
+							'{{p}}Click on "Enable SFTP Credentials". Once you’ve enabled SFTP, you’ll see your SFTP URL, Port Number, Username and Password. Keep these credentials safe!{{/p}}' +
+								'{{p}}The username and password are generated by the system automatically. These are unique to your site, so if you have multiple sites, you’ll need to use multiple usernames and passwords, one for each site, in your SFTP Client.{{/p}}' +
+								'{{p}}After you’ve created a user and password, you can enter them in an SFTP Client of your choice! If you don’t have a preference, {{a}}Filezilla{{/a}} is a popular and free SFTP client. Clicking the copy button will copy them to your clipboard for ease of use.{{/p}}',
+							{
+								components: {
+									p: <p />,
+									a: <a href={ FILEZILLA_URL } />,
+								},
+							}
+						) }
+					</Accordion>
+				</div>
+			) }
+			{ displayQuestionsAndButton && (
+				<>
+					<p className="sftp-card__enable-warning">
+						{ translate(
+							'{{strong}}Ready to access your website files?{{/strong}} Keep in mind, if mistakes happen you can restore your last backup, but will lose changes made after the backup date.',
+							{
+								components: {
+									strong: <strong />,
+								},
+							}
+						) }
+					</p>
+					<Button onClick={ createUser } primary>
+						{ translate( 'Enable SFTP Credentials' ) }
+					</Button>
+				</>
+			) }
 			{ ( username || disabled ) && (
 				<table
 					className={ classNames( 'sftp-card__info-table', { [ 'is-placeholder' ]: disabled } ) }
 				>
 					<tbody>
-						{ map( sftpData, ( data, title ) => (
-							<tr key={ title }>
-								<th>{ title }:</th>
-								<td>
-									<span>{ ! disabled && data }</span>
-								</td>
-							</tr>
-						) ) }
+						<tr key={ translate( 'URL' ) }>
+							<th>{ translate( 'URL' ) }:</th>
+							<td>
+								<>
+									<span>{ ! disabled && SFTP_URL }</span>
+									<ClipboardButton text={ SFTP_URL } onCopy={ () => setIsCopied( 'url' ) } compact>
+										{ urlIsCopied ? translate( 'Copied!' ) : translate( 'Copy URL' ) }
+									</ClipboardButton>
+								</>
+							</td>
+						</tr>
+						<tr key={ translate( 'Port' ) }>
+							<th>{ translate( 'Port' ) }:</th>
+							<td>
+								<>
+									<span>{ ! disabled && SFTP_PORT }</span>
+									<ClipboardButton
+										text={ SFTP_PORT.toString() }
```

Comment:
ClipboardButton didn't like a number being passed into the text prop - worked, but threw a console error - I fixed this.